Please check requirements.txt for PyWin version

ERROR: Could not find a version that satisfies the requirement pywin32==306 (from versions: none)
ERROR: No matching distribution found for pywin32==306
Error: Process completed with exit code 1.

--- a/requirements.txt
+++ b/requirements.txt
@@ -464,7 +464,7 @@ pywavelets==1.4.1
     # via scikit-image
 pywebio==1.8.2
     # via -r requirements.in
-pywin32==306
+pywin32==305
     # via jupyter-core
 pywinpty==2.0.11
     # via

